index.html

```diff
@@ -38,10 +38,10 @@ <h1 align="center">Mega Todo</h1>
             <td><input type="text" name="" id="" class="estimatedTimeInput"></td>
             <td>
                 <select id="status" class="status">
-                    <option value="Complete">Complete
+                    <option value="Not Started">Not Started
                     <option value="Working">Working On
                     <option value="Just Started">Just Started
```

Comment:
```suggestion
                    <option value="Just Started">Just Started
                    <option value="Working">Working On
```

Try committing this 👍